quipucords/api/common/entities.py

```diff
@@ -309,10 +309,11 @@ def _get_deployment_report(cls, report_id):
             .filter(pk=report_id)
             .get()
         )
-        # openshift sources won't make sense in insights reports.
+        # openshift/ansible sources won't make sense in insights reports.
         fingerprints = list(
             SystemFingerprint.objects.filter(deployment_report=deployment_report)
             .exclude(sources__icontains=DataSources.OPENSHIFT)
+            .exclude(sources__icontains=DataSources.ANSIBLE)
```

Comment:
Just a thought for the future, _not_ something to change now: Do you foresee this being a pattern as we expand source types in the not-too-distant future? At some point, would it be better to interrogate some kind of class metadata like some kind of "relevant for deployment reports" flag? Or is that _too_ forward thinking and possibly overkill?